protx-client/src/components/ProTx/components/charts/MainChart.jsx

```diff
@@ -195,21 +197,51 @@ function MainChart({ data, showInstructions }) {
       }
 
       const plotState = protxMaltreatmentDistribution.data;
+      const countyName = getSelectedGeographyName(
+        selection.geography,
+        selection.selectedGeographicFeature
+      );
+      const plotDetailSectionTitle = selection.selectedGeographicFeature ? (
+        <>
+          {getSelectedGeographyName(
+            selection.geography,
+            selection.selectedGeographicFeature
+          )}{' '}
+          County
+          <Button
+            className={styles.link}
+            color="link"
+            onClick={() => setShowAgeModal(true)}
+          >
+            <MaltreatmentAgeBreakdown
+              isOpen={showAgeModal}
+              toggle={() => setShowAgeModal(false)}
+              geographyLabel={countyName + ' County'}
+              geography={selection.geography}
+              selectedGeographicFeature={selection.selectedGeographicFeature}
+            />
+            View County Maltreatment Percentages by Age
+          </Button>
+        </>
+      ) : (
+        ''
+      );
 
       return (
-        <div className="maltreatment-chart">
-          <div className="maltreatment-types-plot">
-            <div className="maltreatment-types-plot-layout">
-              <MaltreatmentDetails
-                geography={selection.geography}
-                selectedGeographicFeature={selection.selectedGeographicFeature}
-                maltreatmentTypes={selection.maltreatmentTypes}
-                data={data}
-              />
-              <MainPlot plotState={plotState} />
-              <ChartInstructions currentReportType="hidden" />
-            </div>
-          </div>
+        <div className={styles['main-chart']}>
+          <span className={styles['main-chart-title']}>
+            <span className={styles['main-chart-title-text']}>
+              {plotDetailSectionTitle}
```

Comment:
suggestion: I would consider just placing the contents of plotDetailSectionTitle directly here and avoid using that variable.

So something like:
```
return (
  <div className={styles['main-chart']}>
    <span className={styles['main-chart-title']}>
      <span className={styles['main-chart-title-text']}>
        <span className={styles['main-chart-title-text']}>
            {selection.selectedGeographicFeature &&
                  {countyName} County
                  <Button
                    className={styles.link}
                    color="link"
                    onClick={() => setShowAgeModal(true)}
                  >
             ...
    }

```